Formula/guile.rb

```diff
@@ -48,6 +49,15 @@ class Guile < Formula
     cause "Segfaults during compilation"
   end
 
+  # http://debbugs.gnu.org/cgi/bugreport.cgi?bug=23870
+  # https://github.com/Homebrew/homebrew-core/issues/1957#issuecomment-229347476
+  if MacOS.version >= :el_capitan
```

Comment:
Does fail to build without:

```
Undefined symbols for architecture x86_64:
  "_clock_getcpuclockid", referenced from:
      _scm_init_stime in libguile_2.0_la-stime.o
ld: symbol(s) not found for architecture x86_64
```

However, with the patch it instead fails at runtime because it tries to reference `clock_gettime` and it isn't available in 10.11's actual `libSystem`.